packages/widgets/i18n/en.pot

```diff
@@ -55,3 +55,20 @@ msgstr ""
 
 msgid "Could not load children"
 msgstr ""
+
+msgid "Number of items per page"
+msgstr ""
+
+msgid ""
+"Viewing page {{page}} of {{pageCount}}, item {{firstItem}}-{{lastItem}} of "
+"{{total}}"
```

Comment:
I suggest removing the "Viewing page 1 of 10...", I'm not sure that is really that relevant. It could also be displayed as the selected item in the 'go-to page' select if it is important (which I think is already being done with `availablePages`?).

So this could then be reworked to "25–50 of 300"

Reply:
We do lose some information when we get rid of this text. The current page can be seen directly in the dropdown, but information about the total number of pages is hidden. The user would have to open the "Go to page" select and see the last option to know how many pages there are.

Plus, the "Go to page" select should be an optional element, so we can't rely on it to be a unique source of information.

I think my preference would be to keep this information but just reduce the character-count a little bit. How about:
`Page 1 of 10, item 25-50 of 300`?

Reply:
Yeah, that's a good point that the select might not always be used. I can't think of any usecase where it's really useful to know the page number, but I'm sure there are out there. I agree with your conclusion and suggestion of `Page 1 of 10, items 25-50 of 300`.